examples/combobox/grid-combo.html

```diff
@@ -77,13 +76,13 @@ <h2 id="ex_label">Example</h2>
       </div>
```

Comment:
Suggest adding `aria-readonly="true"` to the grid, to prevent each individual grid cell announcing it is editable (as happens in NVDA + FF)

Reply:
This is an NVDA bug. It would not align with the ARIA spec to add aria-readonly. There is actually no such thing as a editable grid. You can have an editable gridcell in a spreadshett type application where the cell content is directly editable, but the grid itself is never editable. This is explained in the spec.